src/backend/marsha/markdown/factories.py

```diff
@@ -36,6 +36,7 @@ class MarkdownDocumentFactory(DjangoModelFactory):
 
     class Meta:  # noqa
         model = models.MarkdownDocument
+        skip_postgeneration_save = True
```

Comment:
FMI, why adding this ?

Reply:
There is a deprecation warning about this feature https://github.com/FactoryBoy/factory_boy/blob/63c2df43ea640adb08d68c2b09c542667691f588/factory/django.py#L181